include/zephyr/bluetooth/conn.h

```diff
@@ -1774,6 +1774,66 @@ struct bt_br_conn_param {
 struct bt_conn *bt_conn_create_br(const bt_addr_t *peer,
 				  const struct bt_br_conn_param *param);
 
+/** @brief Life-span states of SCO channel. Used only by internal APIs
+ *  dealing with setting channel to proper state depending on operational
+ *  context.
+ */
+enum bt_sco_state {
```

Comment:
If it's only used by internal APIs, could it be moved to `conn_internal.h`?
If it's designed to be used from an upper layer (possibly not in the zephyr tree), this line needs to be reworded.

Reply:
Due to the new parameter `struct bt_sco_chan *chan` is added to function `bt_conn_create_sco`, this enum needs to be defined in same file `conn.h`.
If function `bt_conn_create_sco` could be moved to `conn_internal.h` or `sco_internal.h`, this new added code block could be moved to corresponding file.

Reply:
Okay. After reading your explanation above, I would then prefer the SCO API be internal for now.
Then `conn.h` doesn't need to be modified.

Reply:
All of changes are moved to `sco_internal.h`